app/assets/javascripts/spotlight/pages.js.erb

```diff
@@ -22,8 +22,7 @@ Spotlight.onLoad(function(){
   if (instance.length) {
     var editor = new SirTrevor.Editor({
       el: instance[0],
-      blockTypes:["Heading", "Text", "List","Quote","Iframe", "Video", "Oembed", "Rule", "UploadedItems",
-                 "Browse", "FeaturedPages", "SolrDocuments",  "SolrDocumentsCarousel", "SolrDocumentsEmbed", "SolrDocumentsFeatures", "SolrDocumentsGrid", "SearchResults"],
+      blockTypes: <%= Spotlight::Engine.config.sir_trevor_widgets %>,
```

Comment:
I thought of another approach for this too... we could add this list as a `data-` attribute on the `js-st-instance`. This would have the benefit of not depending on ERB parsing, and open it up to things like exhibit-specific customization (or run-time things like... if there are no solr documents, you don't get those widgets).

:man_shrugging: